mobile/lib/modules/asset_viewer/image_providers/immich_local_image_provider.dart

```diff
@@ -49,18 +50,18 @@ class ImmichLocalImageProvider extends ImageProvider<Asset> {
     StreamController<ImageChunkEvent> chunkEvents,
   ) async* {
     if (_loadPreview) {
-      // TODO: Use local preview
+      // Load a small thumbnail
+      final thumbBytes = await asset.local?.thumbnailDataWithSize(
+        const ThumbnailSize.square(2000),
+        quality: 100,
+      );
+      if (thumbBytes == null) {
+        throw StateError("Loading thumb for ${asset.fileName} failed");
```

Comment:
No need to error. We can simply try the original still..?

Reply:
Good point.